include/vrv/staff.h

```diff
@@ -144,11 +144,6 @@ class Staff : public Object,
     virtual double GetDrawingRotate() const;
     ///@}
 
-    /**
-     * Adjust drawingStaffSize based on rotate angle
-     */
-    void AdjustDrawingStaffSize();
-
```

Comment:
Do you really need to remove this? I think this is use outside neon

Reply:
Oops, I'll revert it then